### PR TITLE
Feature/documental improvements

### PR DESCRIPTION
**Pull request by smartsena**

### CODE REVIEW

File: documental/admin.py [hunk 1]
```diff
@@ -34,23 +35,23 @@ class DocumentalDocsAdmin(admin.ModelAdmin):
...
-        'path_documento',
-        'no_documento',
-        'usuario_id',
-        'st_disponivel',
-        'st_excluido',
-        'dt_cadastro',
-        'dt_atualizacao',
+        'id_document',
+        'path_document',
+        'no_document',
```
Comment (guigawiller) [inline documental/admin.py:40]: Trailing whitespace.

Please use pep8 to format this file.

File: documental/filters.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ class DocumentalDocsFilter(rest_framework.FilterSet):
     """
```
Comment (guigawiller) [inline documental/filters.py:34]: Trailing whitespace.

Please use pep8 to format this file.

File: documental/models.py [hunk 3]
```diff
@@ -5,26 +5,32 @@
```
Comment (guigawiller) [inline documental/models.py:5]: Trailing whitespace.

Please use pep8 to format this document.

File: documental/models.py [hunk 4]
```diff
@@ -55,65 +59,66 @@ class Usuario(models.Model):
...
 
 class DocumentalDocs(models.Model):
     """DocumentalDocs model data for documental model."""
 
-    id = models.IntegerField(
-        _('Primary key'),
-        unique=True,
-        primary_key=True,
+    id_document = models.IntegerField(
+        #Estudar viabilidade construir chave composta id_document+action_type
```
Comment (guigawiller) [inline documental/models.py:74]: Remove this comment.

File: documental/models.py [hunk 5]
```diff
@@ -55,65 +59,66 @@ class Usuario(models.Model):
     )
     
     class Meta:
-        """"Meta class for `documental.Usuario` model."""
+        """"Meta class for `documental.UsersCMR` model."""
```
Comment (guigawiller) [inline documental/models.py:62]: Put blank line after docstrings.

File: documental/models.py [hunk 6]
```diff
@@ -189,4 +194,4 @@ def __str__(self) -> str:
         Returns:
             str: model data path_documento.
         """
```
Comment (guigawiller) [inline documental/models.py:196]: Put blank line after docstrings.

File: documental/serializers.py [hunk 7]
```diff
@@ -13,9 +13,10 @@ class Meta:
         """Meta class for `ActionListSerializers` serializer."""
```
Comment (guigawiller) [inline documental/serializers.py:13]: Put blank line after docstrings.

File: documental/serializers.py [hunk 8]
```diff
@@ -24,9 +25,9 @@ class UsuarioSerializers(serializers.ModelSerializer):
 
     class Meta:
         """Meta class for `UsuarioSerializers` serializer."""
```
Comment (guigawiller) [inline documental/serializers.py:27]: Put blank line after docstrings.

File: documental/serializers.py [hunk 9]
```diff
@@ -37,28 +38,28 @@ class MapasUsoOcupacaoSoloSerializers(serializers.ModelSerializer):
     Data only for the action category linked to USO_OCUPAÇÃO_DO_SOLO
     """
 
-    usuario_id = UsuarioSerializers()
-    id_acao = ActionListSerializers()
+    usercmr_id = UsuarioSerializers()
+    action_id = ActionListSerializers()
 
     class Meta:
         """Meta class for `MapasUsoOcupacaoSoloSerializers` serializer."""
```
Comment (guigawiller) [inline documental/serializers.py:45]: Put blank line after docstrings.

File: documental/serializers.py [hunk 10]
```diff
@@ -80,28 +81,28 @@ class DocumentosTISerializers(serializers.ModelSerializer):
     Data only for the action category linked to DOCUMENTAL_TI.
     """
 
-    usuario_id = UsuarioSerializers()
-    id_acao = ActionListSerializers()
+    usercmr_id = UsuarioSerializers()
+    action_id = ActionListSerializers()
 
     class Meta:
         """Meta class for `DocumentosTISerializers` serializer."""
```
Comment (guigawiller) [inline documental/serializers.py:88]: Put blank line after docstrings.

File: documental/serializers.py (hunk 8 above)
Comment (guigawiller) [reply documental/serializers.py:27]: Trailing whitespaces.

Use pep8 to format this file.

File: documental/serializers.py [hunk 11]
```diff
@@ -70,7 +71,7 @@ def to_representation(self, instance):
         """
 
         url_document = super().to_representation(instance)
-        url_document['url_doc'] = urllib.parse.urljoin(settings.DOCUMENTOS, instance.path_documento)     
+        url_document['url_doc'] = urllib.parse.urljoin(settings.DOCUMENTOS, instance.path_document)
```
Comment (guigawiller) [inline documental/serializers.py:74]: Long line.

File: documental/serializers.py [hunk 12]
```diff
@@ -113,5 +114,5 @@ def to_representation(self, instance):
         """
 
         url_document = super().to_representation(instance)
-        url_document['url_doc'] = urllib.parse.urljoin(settings.DOCUMENTOS, instance.path_documento)
+        url_document['url_doc'] = urllib.parse.urljoin(settings.DOCUMENTOS, instance.path_document)
```
Comment (guigawiller) [inline documental/serializers.py:117]: Long line.

File: documental/views.py [hunk 13]
```diff
@@ -22,7 +22,7 @@ class AuthModelMix:
 class ActionListView(AuthModelMix, generics.ListAPIView):
     """Returns the list of data in `models.DocsAction`."""
```
Comment (guigawiller) [inline documental/views.py:24]: Trailing whitespaces.

Use pep8 to format this document.

**Review by guigawiller — CHANGES_REQUESTED** [review]
Please use pep8 to format the code.

File: documental/filters.py (hunk 2 above)
Comment (smartsena) [reply documental/filters.py:34]: Removed.

File: documental/admin.py (hunk 1 above)
Comment (smartsena) [reply documental/admin.py:40]: Removed.

File: documental/models.py (hunk 3 above)
Comment (smartsena) [reply documental/models.py:5]: Adjusted.

File: documental/models.py (hunk 4 above)
Comment (smartsena) [reply documental/models.py:74]: Removed.

File: documental/models.py (hunk 5 above)
Comment (smartsena) [reply documental/models.py:62]: Removed.

File: documental/models.py (hunk 6 above)
Comment (smartsena) [reply documental/models.py:196]: Removed.

File: documental/serializers.py (hunk 11 above)
Comment (smartsena) [reply documental/serializers.py:74]: Adjusted.

File: documental/serializers.py (hunk 9 above)
Comment (smartsena) [reply documental/serializers.py:45]: .

File: documental/serializers.py (hunk 10 above)
Comment (smartsena) [reply documental/serializers.py:89]: .

File: documental/views.py (hunk 13 above)
Comment (smartsena) [reply documental/views.py:24]: .

File: documental/serializers.py (hunk 8 above)
Comment (smartsena) [reply documental/serializers.py:27]: .

File: documental/serializers.py (hunk 7 above)
Comment (smartsena) [reply documental/serializers.py:13]: .

**Review by guigawiller — APPROVED** [review]

**Review by guigawiller — APPROVED** [review]

**Pull request merged**